Source/WebCore/style/StyleBuilderConverter.h

```diff
@@ -2190,6 +2194,36 @@ inline TimelineScope BuilderConverter::convertTimelineScope(BuilderState&, const
     }) };
 }
 
+inline SingleTimelineRange BuilderConverter::convertAnimationRange(BuilderState& state, const CSSValue& value, TimelineRangeStartOrEnd startOrEnd)
+{
+    if (auto* primitiveValue = dynamicDowncast<CSSPrimitiveValue>(value)) {
+        if (SingleTimelineRange::isOffsetValue(*primitiveValue)) {
+            // <length-percentage>
+            return { SingleTimelineRange::Name::Omitted, convertLength(state, *primitiveValue) };
+        }
+        // <timeline-range-name> or Normal
+        return { SingleTimelineRange::timelineName(primitiveValue->valueID()), (startOrEnd == TimelineRangeStartOrEnd::Start ? WebCore::Length(0, LengthType::Percent) : WebCore::Length(100, LengthType::Percent)) };
```

Comment:
You could probably drop `WebCore::` from Length?

Reply:
The compiler gets confused with WebCore::Style::Length here if I don't include it